lib/src/coap_client.dart

```diff
@@ -305,8 +305,9 @@ class CoapClient {
   /// Discovers remote resources.
   Future<Iterable<CoapWebLink>?> discover({
     final String query = '',
+    final bool confirmable = true,
```

Comment:
Not sure if `discover` _can_ be non-confirmable. I'm thinking the response will be a piggybacked ack.

Californium also only seems to support the query argument for the discover method: https://github.com/eclipse/californium/blob/3eecb4c739a40ac1af066a89d863be338fa54e00/californium-core/src/main/java/org/eclipse/californium/core/CoapClient.java#L559

Reply:
Good catch, I think that might have been a copy-and-paste mistake ;)